syntax = "proto3";

// input image, width, height
message Frame {
    int32 id = 1; 
    int32 id_slot = 2;
    string b64image = 3;
    int32 width = 4;
    int32 height = 5;
    float creation_timestamp = 6;
}

// service
service FrameProcedure {
    rpc HandleNewFrame(Frame) returns (Empty) {}
}


